utils/setup/test_pip_packages.sh

```diff
@@ -3,11 +3,10 @@
 venv_dir=`mktemp -u -d`
 python3.7 -m venv ${venv_dir}
 . ${venv_dir}/bin/activate
-
-# Install our requirements
 pip install --upgrade pip
-pip install -r requirements.txt
 
-# Install our package on top
+# Install minimum packages
 pip install .[test,train,camera-obs]
```

Comment:
I would not say these are minimum packages but they are definitely the test suite packages.

Reply:
Rewrote the comment.